test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -174,6 +176,7 @@
       app
       [jetty9-service/jetty9-service
        webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
        status-service]
       (merge status-service-config
         {:web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/alternate-status"}})
```

Comment:
Do you think it would be manageable to add an assertion into one or more of these tests to validate that the logging was actually happening if the config is set up properly?

I was going to suggest maybe also adding a test to validate that everything starts up properly if the new config is missing, but this test probably implicitly covers that for now?